Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Image-Cropper/index.md

```diff
@@ -64,17 +64,28 @@ For rendering a cropped media item, the `.GetCropUrl` is used:
 ### MVC View Example to output a "banner" crop from a cropper property with the alias "image"
 
 ```html
-<img src="@Url.GetCropUrl(Model.Image, "banner")" />
+<img src="@(Url.GetCropUrl(Model.Image, "banner"))" />
 ```
 
 Or, alternatively:
+```html
+<img src="@(Model.Image.GetCropUrl("banner", Current.ImageUrlGenerator))" />
+```
 
+This was obsoleted in 8.7.0:
 ```html
-<img src="@Model.Image.GetCropUrl("banner")" />
+<img src="@(Model.Image.GetCropUrl("banner"))" />
```

Comment:
Which os these two snippets are now obsolete?
And wouldn't it make sense to simply remove the one that doesn't apply to the current version anymore? 🤔

Reply:
I wasn't sure about marking it as obsoleted, am more than happy to remove and have done so :-)